src/components/application_manager/rpc_plugins/vehicle_info_plugin/include/vehicle_info_plugin/vehicle_info_plugin.h

```diff
@@ -49,7 +49,7 @@ class VehicleInfoPlugin : public plugins::RPCPlugin {
             app_mngr::rpc_service::RPCService& rpc_service,
             app_mngr::HMICapabilities& hmi_capabilities,
             policy::PolicyHandlerInterface& policy_handler,
-            resumption::LastState& last_state) OVERRIDE;
+            resumption::LastStateWrapperPtr last_state) OVERRIDE;
```

Comment:
re-add and deprecate old signature

Reply:
@jacobkeeler Done in 4fdae6f97e8445b4aebe5269c4163e028115f508